packages/uno-preset/src/theme.ts

```diff
@@ -36,6 +36,7 @@ export const extendTheme: ThemeExtender<Theme> = (baseTheme) => ({
 
   // colors
   colors: {
+    ...baseTheme.colors,
```

Comment:
Nice 👍 

We don't need any of the colors below now, they're [included in the `baseTheme`](https://github.com/unocss/unocss/blob/main/packages/preset-mini/src/_theme/colors.ts#L10-L14)